dts/arm/silabs/efr32fg1p.dtsi

```diff
@@ -173,6 +173,11 @@
 
 	};
 
+	bt_hci_silabs: bt_hci_silabs {
+		compatible = "silabs,bt-hci";
+		status = "okay";
+	};
```

Comment:
As mentioned by chat, none of the `efr32fg*` support Bluetooth.

Reply:
> As mentioned by chat, none of the `efr32fg*` support Bluetooth.

I've tried to keep this PR 1:1 with the existing enablement of the drivers through Kconfig. So if a driver would get enabled before this PR for a certain platform, then it would get enabled also after this PR. I could do this kind of fixes also as part of this PR, but I have a feeling it'd be better to do it separately. It's possible (maybe even likely) that this PR will only go in after the next release, however these incorrect indications of Bluetooth support should probably be fixed before the release.

Reply:
I believe the condition `depends on SOC_SERIES_EFR32BG22 || SOC_SERIES_EFR32MG24 || SOC_SERIES_EFR32BG27` prevented instantiation of HCI driver on wrong boards.

Maybe we should keep this PR as-is for now and prepare a new PR to remove this weird condition.